tests/layouts/test_auto.py

```diff
@@ -45,6 +45,16 @@ def test_layout_umap_int_node_ids(self):
 
         self.assertEqual(len(node_positions), len(graph.nodes()))
 
+    def test_layout_umap_directed(self):
```

Comment:
can you add the same test but for an unweighted network?

Reply:
just added!